Makefile

```diff
@@ -319,23 +319,6 @@ endef
 generate-k8s-api: ## Generate Cilium k8s API client, deepcopy and deepequal Go sources.
 	$(ASSERT_CILIUM_MODULE)
 
-	$(eval TMPDIR := $(shell mktemp -d))
```

Comment:
Why is this block getting removed?

Reply:
Oh my bad! I don't remember why I removed this :sweat_smile:

Reply:
Probably to resolve some conflict :thinking: